.github/workflows/build.yml

```
@@ -0,0 +1,18 @@
+name: build the package
+on:
+  pull_request:
+jobs:
+  run:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: INstall Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.19.x
+      - name: Checkout code
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```

Any reason not going for v3?

---

updated to v3